### PR TITLE
Add HTTP 451 Unavailable For Legal Reasons status

### PR DESCRIPTION
**Pull request by dakatsuka**
HTTP 451was approved by the IESG on December 18, 2015.
https://en.wikipedia.org/wiki/HTTP_451

### CODE REVIEW

**Comment by bryce-anderson** [comment]
Cool, thanks!
If you don't mind, could you also add some ResponseGenerator's to the DSL to cover it as well? https://github.com/http4s/http4s/blob/master/dsl/src/main/scala/org/http4s/dsl/package.scala

**Comment by dakatsuka** [comment]
Added :+1:

**Comment by bryce-anderson** [comment]
Sorry to be a nag, but I think it could still take a generator syntax: https://github.com/http4s/http4s/blob/master/dsl/src/main/scala/org/http4s/dsl/package.scala#L79.
Sorry for not being more specific in my previous comment. If you've exhausted your time I could add it myself after merging this.

**Comment by bryce-anderson** [comment]
Awesome, thank you. I'll merge this as soon as travis gives the green light.

File: dsl/src/main/scala/org/http4s/dsl/package.scala [hunk 1]
```diff
@@ -128,6 +129,7 @@ package object dsl extends Http4s {
   implicit class PreconditionRequiredSyntax(val status: PreconditionRequired.type) extends AnyVal with EntityResponseGenerator
   implicit class TooManyRequestsSyntax(val status: TooManyRequests.type) extends AnyVal with EntityResponseGenerator
   implicit class RequestHeaderFieldsTooLargeSyntax(val status: RequestHeaderFieldsTooLarge.type) extends AnyVal with EntityResponseGenerator
+  implicit class UnavailableForLegalReasons(val status: UnavailableForLegalReasons.type) extends AnyVal with EntityResponseGenerator
```
Comment (bryce-anderson) [inline dsl/src/main/scala/org/http4s/dsl/package.scala]: Looks like you need to add a 'Syntax' suffix here or we have a symbol name issue.

**Comment by dakatsuka** [comment]
:bow:

**Pull request merged**